src/Impostor.Server/Events/Game/Player/PlayerChatEvent.cs

```diff
@@ -22,5 +23,7 @@ public PlayerChatEvent(IGame game, IClientPlayer clientPlayer, IInnerPlayerContr
         public IInnerPlayerControl PlayerControl { get; }
 
         public string Message { get; }
+
+        public bool IsCancelled { get; set; }
```

Comment:
Does this cancel the entire packet or is the RPC removed

Reply:
Cancels forwarding of the packet, what would you modify?

Reply:
Take out the chat message from the gamedata RPC in the case there are more gamedata updates but I guess there can't be